config/manager/manager.yaml

```diff
@@ -44,7 +44,7 @@ spec:
           type: RuntimeDefault
       containers:
       - command:
-        - /usr/local/bin/manager
+        - /usr/local/bin/siteconfig-manager
```

Comment:
How did this work before :confused:

Reply:
I updated this in my last [PR](https://github.com/stolostron/siteconfig/pull/144/commits/3b224f5768ac7e30a76d3d7ab01540e190d2290a#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557) and missed updating here.